layouts/partials/parts-tools.en.md

```diff
@@ -0,0 +1,27 @@
+## Tools
+
+These tools are needed to assemble the sensor. The
+most specialized tool is the crimping tool, which your local
+Maker Space / Fab Lab surely has lying around. You may not need to buy each 
+of these tools if you can borrow one.
+
+* [Crimping Pliers](https://www.amazon.de/gp/product/B07VX6YGQ8) for JST-XH
+  connectors (other brands or types of pliers that are compatible with these 
+  connectors will also work, e.g., Engineer PA-09).
+* A good, adjustable soldering iron or soldering station with an appropriate 
+  solution for venting
```

Comment:
Maybe "ventilation" is more appropriate here? I'm not a native English speaker, but I believe ventilation is more about removing the fumes from the work area and having fresh air to breathe, while venting is about moving gases outside a building or container.... I think :joy:

Reply:
For this, I think it's splitting hairs a little. I could go either way, but I'm not married to anything. It would probably be better as "an appropriate ventilation solution" or "an appropriate solution to handle the soldering vapors." What would you prefer?